src/data/data-files/aws-regions.json

```diff
@@ -1,282 +0,0 @@
-[
```

Comment:
Are we sure we want to do this cleanup now? Would be this conflict with upstream?

Reply:
Yea, we should probably manage it separately.  The files arent needed, but probably shouldnt be coupled with this PR